migration-config.conf.sample

```diff
@@ -38,6 +38,9 @@ new_tenant_name_format = "%(original)s-Migrated"
 new_secgroup_name_format = "%(original)s-Migrated"
 new_subnet_name_format = "%(original)s-Migrated"
 new_network_name_format = "%(original)s-Migrated"
+new_user_name_format = "%(original)s-Migrated"
```

Comment:
Could you please just leave the default as `%(original)s` (there's no real reason someone would want to have a different username, but having the option is nice, so the best route would be to just make the default case have the same name)